packages/recommend-core/src/utils/mapToTrendingFacets.ts

```diff
@@ -0,0 +1,31 @@
+import { MultipleQueriesResponse } from '@algolia/client-search';
+
+import { TrendingFacet } from '../types';
+
+import { sortBy } from './sortBy';
+
+type MapToRecommendations<TObject> = {
+  response: MultipleQueriesResponse<TObject>;
+  maxRecommendations?: number;
+};
+
+export function mapToTrendingFacets<TObject>({
```

Comment:
Could we avoid this duplicated logic (with `mapToRecommendations`)? It'd be possible by accepting `hits` instead of `response` and using `uniqBy` one call higher.

Reply:
I tried what you are saying. And it's working, Thank you for your advice :) (If I understood correctly)